pkg/secrets/k8s/k8s_secrets_test.go

```diff
@@ -41,6 +41,26 @@ func TestKubernetesSecrets(t *testing.T) {
 			})
 		})
 
+		Convey("Given a map of data entries and a secret with backslashes", func() {
+			m := make(map[string][]byte)
```

Comment:
can we give `m` a proper name? what does it mock? i think we have this in several places so please fix that too

Reply:
fixed in this file and in codebase